### PR TITLE
chore: add script to run cli in playground context

### PR DESCRIPTION
**Pull request by warflash**
Trying to get familiar with the CLI at the moment and couldn't find an easy way to use the actual local CLI build for the playground.
Very well possible I just missed the obvious and existing way to go about that process 😅

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "dev:bun": "bun --bun ./bin/nuxi.mjs dev ./playground",
     "lint": "eslint . && prettier --check src",
     "lint:fix": "eslint --fix . && prettier --write src",
+    "play:nuxi": "cd ./playground && node ../bin/nuxi.mjs",
```
Comment (pi0) [inline package.json:30]:
```suggestion
    "play": "./bin/nuxi.mjs --cwd ./playground",
```

Comment (warflash) [reply package.json:30]: Thanks!
Played with that a bit and it seems like we need to pass commands before the --cwd flag, so e.g.
`node ./bin/nuxi.mjs analyze --cwd ./playground`, but then we loose the ability to feed the command we want at the console level.
With the original suggestion you were able to execute `pnpm play cleanup` or any other nuxi command directly.
Hope that makes sense

Comment (pi0) [reply package.json:30]: Yeah thanks for checking. I think there is area to improve for this to make it possible.

Comment (rohrig) [reply package.json:30]: @warflash 
I had the same struggle when first playing with the cli locally. Turns out the simplest solution I could find was to run it like so:

from the playground dir:

```bash
    ../bin/nuxi.mjs <whatever normally come after running nuxi>
```

Comment (pi0) [reply package.json:30]: Yes, that's what i also normally do today. I think point of this PR, (which is also valid for real projects) is to prepare a generic command that allows nuxi operating in separate directory. There are few different ways such as `cd` to the dir or directly `nuxi <dev> [dir]` (limitation is, some command only accept `--dir`)

**Comment by pi0** [comment]
Tracking via https://github.com/nuxt/cli/issues/365

**Pull request closed without merging**